src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
```

Comment:
You wrote it in a lean form, but this will not always be the best option, especially with regard to readability. Separating the result of each method into variables would make it easier to understand what it is doing.